include/conversation.php

```diff
@@ -144,222 +144,106 @@ function localize_item(&$item)
 		$item['body'] = item_redir_and_replace_images($extracted['body'], $extracted['images'], $item['contact-id']);
 	}
 
-	/*
-	heluecht 2018-06-19: from my point of view this whole code part is useless.
-	It just renders the body message of technical posts (Like, dislike, ...).
-	But: The body isn't visible at all. So we do this stuff just because we can.
-	Even if these messages were visible, this would only mean that something went wrong.
-	During the further steps of the database restructuring I would like to address this issue.
-	*/
-
-	$activity = DI::activity();
-
-	$xmlhead = "<" . "?xml version='1.0' encoding='UTF-8' ?" . ">";
-	if ($activity->match($item['verb'], Activity::LIKE)
-		|| $activity->match($item['verb'], Activity::DISLIKE)
-		|| $activity->match($item['verb'], Activity::ATTEND)
-		|| $activity->match($item['verb'], Activity::ATTENDNO)
-		|| $activity->match($item['verb'], Activity::ATTENDMAYBE)) {
-
-		$fields = ['author-link', 'author-name', 'verb', 'object-type', 'resource-id', 'body', 'plink'];
-		$obj = Item::selectFirst($fields, ['uri' => $item['parent-uri']]);
-		if (!DBA::isResult($obj)) {
-			return;
-		}
-
-		$author  = '[url=' . $item['author-link'] . ']' . $item['author-name'] . '[/url]';
-		$objauthor =  '[url=' . $obj['author-link'] . ']' . $obj['author-name'] . '[/url]';
-
-		switch ($obj['verb']) {
-			case Activity::POST:
-				switch ($obj['object-type']) {
-					case Activity\ObjectType::EVENT:
-						$post_type = DI::l10n()->t('event');
-						break;
-					default:
-						$post_type = DI::l10n()->t('status');
-				}
-				break;
-			default:
-				if ($obj['resource-id']) {
-					$post_type = DI::l10n()->t('photo');
-					$m = [];
-					preg_match("/\[url=([^]]*)\]/", $obj['body'], $m);
-					$rr['plink'] = $m[1];
-				} else {
-					$post_type = DI::l10n()->t('status');
-				}
-		}
-
-		$plink = '[url=' . $obj['plink'] . ']' . $post_type . '[/url]';
-
-		$bodyverb = '';
-		if ($activity->match($item['verb'], Activity::LIKE)) {
-			$bodyverb = DI::l10n()->t('%1$s likes %2$s\'s %3$s');
-		} elseif ($activity->match($item['verb'], Activity::DISLIKE)) {
-			$bodyverb = DI::l10n()->t('%1$s doesn\'t like %2$s\'s %3$s');
-		} elseif ($activity->match($item['verb'], Activity::ATTEND)) {
-			$bodyverb = DI::l10n()->t('%1$s attends %2$s\'s %3$s');
-		} elseif ($activity->match($item['verb'], Activity::ATTENDNO)) {
-			$bodyverb = DI::l10n()->t('%1$s doesn\'t attend %2$s\'s %3$s');
-		} elseif ($activity->match($item['verb'], Activity::ATTENDMAYBE)) {
-			$bodyverb = DI::l10n()->t('%1$s attends maybe %2$s\'s %3$s');
-		}
-
-		$item['body'] = sprintf($bodyverb, $author, $objauthor, $plink);
-	}
-
-	if ($activity->match($item['verb'], Activity::FRIEND)) {
-
-		if ($item['object-type']=="" || $item['object-type']!== Activity\ObjectType::PERSON) return;
```

Comment:
This all has been killed? Is that correct? Had it been moved or is it dead code?